index.html

```diff
@@ -191,5 +213,46 @@ <h3 class="lets">LETS CONNECT</h3>
             </div>
         </div>    
     </section>
+
+    <section class="contac">
+        <div class="fond">
+            <img src="images/Contactme.jpg" alt="">
+        </div>
+        <div>
+            <h1 class="contacme">Contac me</h1>
+            <p class="textt">If you have an application you are interested in developing, 
+                a feature that you need built or a project taht needs coding. i d love to help with it.
+            </p>
+        </div>
+        <div class="for">
+            <form action="https://formspree.io/f/xknegzyn" method="post">
```

Comment:
- [ ] Only the message from the form gets submitted, together with the message it should also submit the name and email
![image](https://user-images.githubusercontent.com/62705947/187305339-94b56ee6-a275-4a50-921f-27dfa90c93b0.png)